manifests/gcp_marketplace/guide.md

```diff
@@ -43,6 +43,7 @@ gcloud projects add-iam-policy-binding $PROJECT_ID \
   --member=serviceAccount:$SA_NAME@$PROJECT_ID.iam.gserviceaccount.com \
   --role=roles/storage.admin
 
+# Note that you can not bind multiple roles in one line.
```

Comment:
may not necessary to mention it. Flag parsing can't handle multiple roles which is common sense.
--role= xxx --role= yyy

Reply:
Ack. However user might shop around and attempt to do --role=storage.admin,dataproc.admin,dataflow.admin, and hope the argparse can do the list parsing correctly.